langfuse/Sampler.py

```diff
@@ -26,6 +26,8 @@ def sample_event(self, event: dict):
                 trace_id = event["body"]["trace_id"]
             elif "traceId" in event["body"]:
                 trace_id = event["body"]["traceId"]
+            elif "id" in event["body"]:
+                trace_id = event["body"]["id"]
```

Comment:
Why did you make this change? Above, we get the event.body.id, in case the event is a trace-create type.

Reply:
@maxdeichmann I inspected the event object and I saw my chains didn't have traceId, just an id so I added it as the traceId.. it was showing a warning about it

Reply:
Ok I removed it and it seems fine, I'll commit soon